### DISCUSSION TITLE
Short description in Installation overview

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus.backgroundjobs#106] [General]
In our documentation, we have a page which lists all the components and shows the command that can be used to install a backgroundjob.

I wonder if it would be helpful if  that overview contains a short description (1 or 2 lines) that briefly describes what you can do with that job.  It might be even enough to just create a link to the extended documentation of that component ?

**Comment by stijnmoreels** [comment]
Yes, that would be a good enhancement. Normally, we would guide people directly towards the documentation page, and should the README only for a minimal 'index page' structure, but we can probably not assume that everyone will follow that pattern 😉 .
